src/common/components/elements/FollowUnfollowButton.tsx

```diff
@@ -165,23 +165,14 @@ export const FollowUnfollowButton: FC<FollowUnfollowButtonProps> = ({
 
   const loading = connectTo.status === 'loading' || disconnectTo.status === 'loading';
 
-  return type === 'Follow' ? (
+  return (
     <Button5
```

Comment:
Is there a difference between Button5 here and just the default button component, if not we should prob make that consistent

Reply:
There's a fair bit of stuff going on in Button5, including having a spinner for a loading state.